react-app/src/library/services/vendor-implementations/CyberAcoustics/CyberAcoustics.ts

```diff
@@ -163,18 +163,16 @@ export default class CyberAcousticsService extends VendorImplementation {
           const HIDPermissionTimeout = setTimeout(reject, 30000);
           this.requestWebHidPermissions(async () => {
             this.logger.debug("Requesting web HID permissions");
-            const productId = this.deductProductId(originalDeviceLabel);
```

Comment:
Why did we remove this? Didn't we just merge this in?

Reply:
Ah good catch! That's my bad. These changes were pulled from main since that was where the linting fixes took place. But those changes happened _before_ Dragin made their contribution. Totally missed that but will add it back in now. Thank you!